ios/scripts/reload.sh

```
@@ -679,15 +773,40 @@ reload_device() {
+      exit 1
+    fi
+    if [[ ! -x "$QUEUE_SCRIPT" ]]; then
+      echo "error: iPhone unreachable and $QUEUE_SCRIPT is missing; cannot queue the build" >&2
+      exit 1
+    fi
+    queue_mode=1
+    selected_device_name="(unreachable, queueing for $queued_device_id)"
+    echo "==> iPhone unreachable; the signed build will be QUEUED for auto-install on reconnect"
+  else
+    tab=$'\t'
+    selected_device_id="${selection%%$tab*}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)